clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -382,5 +385,31 @@ private AbstractRequestResponse createUpdateMetadataResponse() {
         return new UpdateMetadataResponse(Errors.NONE.code());
     }
 
+    private AbstractRequest createCreateTopicRequest() {
+        CreateTopicRequest.TopicDetails request1 = new CreateTopicRequest.TopicDetails(
+            3, 5, new HashMap<Integer, List<Integer>>(), new HashMap<String, String>());
 
+        Map<Integer, List<Integer>> replicaAssignments = new HashMap<Integer, List<Integer>>();
```

Comment:
Are you sure those request classes were added after we switched to Java 7? We generally use the diamond operator any new code. And people have been switching old code incrementally.